### PR TITLE
Iryna kazlova/notification for create report and update report

### PR DESCRIPTION
**Pull request by rinchan603**

### CODE REVIEW

File: Elevel/Application/Features/ReportCommands/CreateReportCommand.cs [hunk 1]
```diff
@@ -96,6 +103,12 @@ public async Task<Response> Handle(Request request, CancellationToken cancellati
                     }
 
                     report.CreatorId = question.CreatorId;
+                    
+                    userEmail = (await _userManager.Users.FirstOrDefaultAsync(x => x.Id == question.CreatorId)).Email;
+                    if (string.IsNullOrWhiteSpace(userEmail))
+                    {
+                        throw new NotFoundException($"User with {request.UserId}");
```
Comment (parys) [inline Elevel/Application/Features/ReportCommands/CreateReportCommand.cs:110]: but your exception message is not correct. Could you change message about empty or null email?

File: Elevel/Application/Features/ReportCommands/CreateReportCommand.cs [hunk 2]
```diff
@@ -109,6 +122,12 @@ public async Task<Response> Handle(Request request, CancellationToken cancellati
                     }
 
                     report.CreatorId = audition.CreatorId;
+
+                    userEmail = (await _userManager.Users.FirstOrDefaultAsync(x => x.Id == audition.CreatorId)).Email;
+                    if (string.IsNullOrWhiteSpace(userEmail))
+                    {
+                        throw new NotFoundException($"User with {request.UserId}");
```
Comment (parys) [inline Elevel/Application/Features/ReportCommands/CreateReportCommand.cs:129]: same, user is found

File: Elevel/Application/Features/ReportCommands/UpdateReportCommand.cs [hunk 3]
```diff
@@ -60,7 +65,13 @@ public async Task<Response> Handle(Request request, CancellationToken cancellati
                     throw new NotFoundException(nameof(Report), request.Id);
                 }
 
-                if(request.ReportStatus == ReportStatus.Fixed)
+                var userEmail = (await _userManager.Users.FirstOrDefaultAsync(x => x.Id == report.UserId)).Email;
+                if (string.IsNullOrWhiteSpace(userEmail))
+                {
+                    throw new NotFoundException($"User with {report.UserId}");
```
Comment (parys) [inline Elevel/Application/Features/ReportCommands/UpdateReportCommand.cs:73]: same incorrect message

File: Elevel/Application/Profiles/ReportProfile.cs [hunk 4]
```diff
@@ -16,7 +13,9 @@ public ReportProfile()
             CreateMap<Report, GetReportListQuery.ReportDto>()
                 .ForMember(dest => dest.CoachName,
                     opt => opt.MapFrom(src =>
-                        src.Question.Creator.UserName.Replace("_", " ") ?? src.Audition.Creator.UserName.Replace("_", " ") ?? src.Topic.Creator.UserName.Replace("_", " ")))
+                        src.Question.Creator.UserName.Replace("_", " ") ??
```
Comment (parys) [inline Elevel/Application/Profiles/ReportProfile.cs:16]: it's usually 
=> bla1
?? bla2
?? bla3

File: Elevel/Elevel.Test/ReportCommandTest/CreateReportCommandValidatorTests.cs [hunk 5]
```diff
@@ -0,0 +1,23 @@
+﻿using Xunit;
```
Comment (parys) [inline Elevel/Elevel.Test/ReportCommandTest/CreateReportCommandValidatorTests.cs:1]: ideally to not include into commit unfinished work, you can stage files that want to commit, other files will be omitted and stay on you branch in uncommitted status

File: Elevel/Application/Features/ReportCommands/CreateReportCommand.cs (hunk 1 above)
Comment (rinchan603) [reply Elevel/Application/Features/ReportCommands/CreateReportCommand.cs:110]: Done

File: Elevel/Application/Features/ReportCommands/CreateReportCommand.cs (hunk 2 above)
Comment (rinchan603) [reply Elevel/Application/Features/ReportCommands/CreateReportCommand.cs:129]: Done

File: Elevel/Application/Features/ReportCommands/UpdateReportCommand.cs (hunk 3 above)
Comment (rinchan603) [reply Elevel/Application/Features/ReportCommands/UpdateReportCommand.cs:73]: Done

File: Elevel/Application/Profiles/ReportProfile.cs (hunk 4 above)
Comment (rinchan603) [reply Elevel/Application/Profiles/ReportProfile.cs:16]: Done

File: Elevel/Elevel.Test/ReportCommandTest/CreateReportCommandValidatorTests.cs (hunk 5 above)
Comment (rinchan603) [reply Elevel/Elevel.Test/ReportCommandTest/CreateReportCommandValidatorTests.cs:1]: ok

File: Elevel/Application/Features/ReportCommands/UpdateReportCommand.cs [hunk 6]
```diff
@@ -3,15 +3,17 @@
 using Elevel.Application.Interfaces;
 using Elevel.Domain.Enums;
 using Elevel.Domain.Models;
+using FluentValidation;
```
Comment (parys) [inline Elevel/Application/Features/ReportCommands/UpdateReportCommand.cs:6]: duplications

**Review by parys — APPROVED** [review]

File: Elevel/Application/Features/ReportCommands/UpdateReportCommand.cs (hunk 6 above)
Comment (rinchan603) [reply Elevel/Application/Features/ReportCommands/UpdateReportCommand.cs:6]: Done

**Pull request merged**